source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,684 @@ Filters Builders
+
+   * - ``geoWithinBox()``
+     - Selects documents containing a grid coordinates field that exist within the specified box.
+      
+   * - ``geoWithinPolygon()``
+     - Selects documents containing a grid coordinates field that exist within the specified polygon.
+
+   * - ``geoWithinCenter()``
+     - Selects documents containing a grid coordinates field that exist within the specified circle.
+
+   * - ``geoWithinCenterSphere()``
+     - Selects geometries containing a geospatial data field (GeoJSON or legacy coordinate pairs) that exist within the specified circle, using spherical geometry.
+
+   * - ``geoIntersects()``
+     - Selects geometries that intersect with a GeoJSON geometry. The 2dsphere index supports $geoIntersects.
+
+   * - ``near()``
+     - Returns geospatial objects in proximity to a point. Requires a geospatial index. The 2dsphere and 2d indexes support $near.
+
+   * - ``nearSphere()``
+     - Returns geospatial objects in proximity to a point on a sphere. Requires a geospatial index. The 2dsphere and 2d indexes support $nearSphere.
+
+The following code snippet shows how to use a geospatial filter:
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Filters.*;
+
+   // <MongoCollection setup code here>
+
+   collection.find(<geospatial filter>);
+
+.. example::
+
+   TThe following example creates a filter that selects documents where the
+   ``point`` field contains a GeoJSON Geometry object that falls within the
+   given polygon in the geo_points collection:
+
+   .. code-block:: java
+      
+      import static com.mongodb.client.model.Filters.geoWithin;
+
+      // <MongoCollection setup code here>
+
+      Polygon square = new Polygon(Arrays.asList(new Position(0, 0), 
+                                            new Position(4, 0), 
+                                            new Position(4, 4), 
+                                            new Position(0, 4),
+                                            new Position(0, 0)));
+   
+      Bson geoWithinComparison = geoWithin("point", square));
+      collection.find(geoWithinComparison);
+      List<Document> results = new ArrayList<>();
+      collection.find(bitsComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      {
+         "point": {
+            $geoWithin: {
+               $geometry: {
+                  type: "Polygon",
+                  coordinates: [ [ [ 0, 0 ], [ 4 , 0 ], [ 4, 4 ], [ 0, 4 ], [0 , 0] ] ]
+               }
+            }
+         }
+      }
+
+   The output of the above code snippet should look something like this:
+   
+   .. code-block:: json
+      :copyable: false
+
+      { _id: 13, type: "Point", coordinates: [ 2,2 ] }
+      { _id: 15, type: "Point", coordinates: [ 1,3 ] }
```

Comment:
I don't know the answer offhand, but would these be formatted with no spaces between numbers in the array?